tests/integration_tests/with_db/test_audit_table.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from exasol.python_extension_common.deployment.temp_schema import temp_schema
+
+from exasol.analytics.audit.audit import AuditTable
+from exasol.analytics.audit.columns import AuditColumns
+
+
+@pytest.fixture(scope="session")
+def db_schema(pyexasol_connection):
+    with temp_schema(pyexasol_connection) as db_schema:
+        yield db_schema
+
+
+def test_create_audit_table(pyexasol_connection, db_schema):
+    audit_table = AuditTable(db_schema)
+    pyexasol_connection.execute(audit_table.create)
+    columns = pyexasol_connection.execute(
+        f"DESCRIBE {audit_table.table.fully_qualified}"
```

Comment:
use EXA_ALL_COLUMNS instead of DESCRIBE, then you don't need an in comparison which is always a bit dangerous

Reply:
Ah! [Docs](https://docs.exasol.com/db/latest/sql_references/system_tables/metadata/exa_all_columns.htm?Highlight=EXA_ALL_COLUMNS):
>  All users have access to the table.

Reply:
See next push using EXA_ALL_COLUMNS